content/cloud/labels.md

```diff
@@ -0,0 +1,65 @@
+title = "Labels"
+template = "cloud_main"
+date = "2023-10-30T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/labels.md"
+
+---
+- [Labels Overview](#labels-overview)
+- [Benefits of labels](#benefits-of-labels)
+- [How To Create A Label](#how-to-create-a-label)
+- [How to Delete A Label](#how-to-delete-a-label)
+- [Next Steps](#next-steps) 
+
+# Labels Overview
+
+Spin applications are inherently ephermal resources; therefore, you don't necessarily want other Fermyon Cloud resources (such as NoOps SQL Databases) to share the exact same lifecycle. Furthermore, as you build and prototype your Spin application you may want to connect to other resources without necessarily having to recompile your application, but you still need some construct to explain the relationship between your Spin application and the other resource you need it to connect with. 
+
+This is where Fermyon Cloud `labels` are useful. `labels` are construct that allow developers to establish a relationship between a Spin application and it's dependent Fermyon Cloud resources while still preserving independent lifecycles for the respective resources. `Labels` are application scoped and an indirect reference to a named Fermyon Cloud resource that belongs to the user. Whenever that Spin application needs to access that resource, it will use the `label` as reference. The Spin application does not need to know the true name of the resource it's interacting with. This also means a `label` is well understood by all the components within it's respective application, but it doesn't hold any significance to other applications. 
+
+> At this time, only NoOps SQL DB supports `labels`. However, in the future other Fermyon Cloud resources such as Key Value Store will support `labels` as well. 
+
+## Benefits of Labels
+
+Benefits of the Fermyon Clould `labels` include:
+
+* **Easy Sharing**: Share your Fermyon Cloud resource across applications effortlessly.
+* **Resource Creation Control**: You decide when to create and delete Fermyon Cloud resources.
+* **Flexible Lifecycle Management**: Manage Fermyon Cloud resources independently of your Spin application.
+* **Seamless Cloud Integration**: Cloud experience smoothly integrates with local Spin development, no runtime configuration changes required. 
+* **Component Model Alignment**: Change the data resource your application points to without rebuilding it.
+
+## How To Create A Label
+
+Labels can be created in two different manners, either within the component manifest in `spin.toml` or via `spin cloud` plugin via the `spin cloud link` command. Both approaches are equally valid. The path you choose most likely depends on whether your Spin application needs to be linked to the resource immediately at deployment time. If you know the exact topology of your workload, you'll likely include the label in the component manifest in your `spin.toml` at deployment time.
```

Comment:
> If you know the exact topology of your workload, you'll likely include the label in the component manifest in your `spin.toml` at deployment time.

I'm not sure what this is saying.  A label that _isn't_ mentioned in the manifest is doing nothing.  Even if I _don't_ know the "exact topology of my workload," I still need to reference a label in the manifest if I want to use a database.  The label kind of _abstracts_ the application from the broader (cross-application) topology of the workload.

Reply:
Agreed, I've reworked this paragraph entirely to be more application and link centric